integrationTests/vm/wasm/wasmvm/mock_contracts.go

```diff
@@ -27,6 +30,9 @@ var MockInitialBalance = big.NewInt(10_000_000)
 // WalletAddressPrefix is the prefix of any smart contract address used for testing.
 var WalletAddressPrefix = []byte("..........")
 
+var InitialEsdt = uint64(100)
```

Comment:
missing comments here and on the next line
Also, can we rename the file to mockContracts.go?